Sources/MenuAnimator.swift

```diff
@@ -334,18 +307,18 @@ class MenuInteractiveTransition: NSObject, UIViewControllerInteractiveTransition
             }
         }
         
-        if useFinishingSpringOption {
-            UIView.animate(withDuration: transitionDuration - transitionDuration * Double(currentPercentComplete),
+        if params.useFinishingSpringOption {
```

Comment:
options from options is a good question) 
maybe we might use params for inner options?